wa/framework/target/runtime_config.py

```diff
@@ -939,6 +955,12 @@ def check_target(self):
     def validate_parameters(self):
         pass
 
+    def handle_screen_unlock(self):
+        if 'unlock_screen' in self.config and self.config['unlock_screen']:
+            screen_state = self.target.execute('dumpsys window')
+            if 'mDreamingLockscreen=true' in screen_state:
```

Comment:
Would it be worth adding a helper to devlib to try and detect whether the lockscreen is enabled or not rather than having this contained here?

Reply:
Makes sense. Will do that.